godtools/App/Share/Data/UserAuthentication/AuthenticationProviders/AppleAuthentication+AuthenticationProviderInterface.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  AppleAuthentication+AuthenticationProviderInterface.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 5/16/23.
+//  Copyright © 2023 Cru. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import SocialAuthentication
+import Combine
+
+extension AppleAuthentication: AuthenticationProviderInterface {
+    func getPersistedAccessToken() -> AuthenticationProviderAccessToken? {
+        
+        // TODO: - access token renewal will come through MobileContentAPI?
+        
+        return nil
+    }
+    
+    func authenticatePublisher(presentingViewController: UIViewController) -> AnyPublisher<AuthenticationProviderAccessToken?, Error> {
+        
+        return authenticatePublisher()
+            .map { (response: AppleAuthenticationResponse) in
+                
+                guard
+                    let idToken = response.identityToken,
+                    let givenName = response.fullName?.givenName,
+                    let familyName = response.fullName?.familyName
+                else {
+                    return nil
+                }
+                                
+                return AuthenticationProviderAccessToken.apple(
+                    idToken: idToken,
+                    givenName: givenName,
+                    familyName: familyName
+                )
+            }
+            .eraseToAnyPublisher()
+    }
+    
+    func renewAccessTokenPublisher() -> AnyPublisher<AuthenticationProviderAccessToken, Error> {
+        
+        // TODO: - access token renewal will come through MobileContentAPI?
```

Comment:
Hey @rachaelblue for this let's instead Fail with an Error until refreshing is available.  This way the mobile-content-api won't be called and can fail immediately.